testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1043,6 +1043,19 @@
   $proxy.run(cmd, timeout: proxy_timeout)
 end
 
+When(/^I allow all SSL protocols on the proxy's apache$/) do
+  file = '/etc/apache2/ssl-global.conf'
+  key = 'SSLProtocol'
+  val = 'all -SSLv2 -SSLv3'
+  $proxy.run("grep '#{key}' #{file} && sed -i -e 's/#{key}.*$/#{key} #{val}/' #{file}")
+  $proxy.run("systemctl reload apache2.service")
```

Comment:
I think this reload was causing also some issue and we could need a change to a restart.